user_docs/en/userGuide.md

```diff
@@ -1846,6 +1846,18 @@ If you wish to change the update mirror, press the "Change..." button to open th
 Please note that when using an update mirror, the operator of the mirror has access to all [information sent with update checks](#GeneralSettingsCheckForUpdates).
 Contact the operator of the update mirror for details of their data handling policies to ensure you are comfortable with the way your information will be handled before setting an update mirror.
 
+##### Prevent display from turning off during say all or reading with braille {#preventDisplayTurnOff}
+
+| . {.hideHeaderRow} |.|
+|---|---|
+|Options |Default (Enabled), Disabled, Enabled|
+|Default |Enabled|
```

Comment:
please move this to the bottom of the section as per the latest [coding standards](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/codingStandards.md) for more information.